<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Võ Bích Ngọc k234111406</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Dataset:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDataset">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>50</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(136, 149, 114);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenFile">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(136, 149, 114);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveChart">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(136, 149, 114);</string>
     </property>
     <property name="text">
      <string>Save Chart to HTML file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenChart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(136, 149, 114);</string>
     </property>
     <property name="text">
      <string>Open Chart in Browser</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
